php-fpm/Dockerfile

```diff
@@ -1214,6 +1230,10 @@ RUN if [ ${INSTALL_FAKETIME} = true ]; then \
     echo "/usr/lib/x86_64-linux-gnu/faketime/libfaketime.so.1" > /etc/ld.so.preload \
 ;fi
 
+# Fixes SQLSTATE[08001]: [Microsoft][ODBC Driver 17 for SQL Server]TCP Provider: Error code 0x2746
+# for gvv-api when connecting to TravelStudio MSSQL server
+RUN sed -i -E "s/(CipherString\s*=\s*DEFAULT@SECLEVEL=)2/\11/" /etc/ssl/openssl.cnf
```

Comment:
@therealartz 
hm.. interesting, I am getting this `TCP Provider: Error code 0x2746` from CLI but not from http,
does it mean we should have this workaround in another place too?